MANIFEST.in

```
@@ -7,6 +7,7 @@ include demo/README
 include demo/demo.tls.dh
 include demo/*.py
 include demo/*.sh
+include *.toml
```

Why would `.deepsource.toml` be included here? 

Btw, if it is only `.deepsource.toml` that is included here, can this be a specific file identifier, not a file extension one? It is possible other files in the repo match `.toml`, but haven't been included in the package before.

```suggestion
include .deepsource.toml
```

---

@ShadowJonathan thanks for pointing out .

---

Still, i'm curious why `.deepsource.toml` should be included in the final distribution package, what's the benefit in that?

---

Sure @ShadowJonathan .

.deepsource.toml configuration can use to configure DeepSource analysis settings.

DeepSource helps  automatically find and fix issues in your code during code reviews. This tool looks for anti-patterns, bug risks, performance problems, and raises issues. There are plenty of other issues in relation to Bug Discovery and Anti-Patterns which you would be interested to take a look at.

---

I know what deepsource is, I'm just confused why it would have to be included in the final .tar.gz and .whl files generated when python creates these from the source package.

---

I included the file by mistake in the Manifest since the CI gave a hint to do so.
I have now excluded the file from the Manifest and thus, it would not be bundled in the final generated files (.tar.gz and .whl files)
The .deepsource.toml file is not needed to be bundled in the generated files.

---

Thanks, i just wanted to point that out, this comment thread can be closed, then :+1:

---

Thanks @ShadowJonathan .

---

Oh i'm so sorry, i didn't properly look at the lines you changed (changing it from `include` to `exclude`, or doing `exclude` in the first place, cant tell with the force-pushes you did), sorry for the confusion, the current state looks good :+1: